### DISCUSSION TITLE
Is it possible to use many layouts in table filters?

### DISCUSSION THREAD

**Discussion by danie1net0** [filamentphp/filament#12263] [Help]
### Package

Table builder

### Package Version

v3.2

### How can we help you?

I would like to use different layouts of filters in only one table, with some filters placed above the content and others in a modal. Is it possible?

**Comment by leandrocfe** [comment]
Maybe custom widgets with custom form fields? https://filamentphp.com/docs/3.x/panels/resources/widgets
You can also use tabs https://filamentphp.com/docs/3.x/panels/resources/listing-records#using-tabs-to-filter-the-records

**Reply by danie1net0** [reply]
Using widgets, how can I pass the form values to the table query as a filter?
